config/initializers/10-patches.rb

```diff
@@ -98,6 +77,12 @@ def full_message(attribute, message)
 
 module ActionView
   module Helpers
+    module Tags
+      Base.class_eval do
+        attr_reader :method_name
```

Comment:
@ulferts tbh I can't remember 100% why this was necessary (d4d8826a has a bit more explanation, but commit was from last November).